src/ws-header/authorization.js

```diff
@@ -89,14 +65,13 @@ export class Authorization {
 
   /**
    * Update the tokens and notify the header
-   * @param {Object} params Response parameters containing access and refresh token
+   * @param {Object} params Response parameters containing access token
    * @returns {void}
    * @private
    */
   updateTokens(params) {
     const expires = params.expires_in ? parseInt(params.expires_in, 10) : 3600;
     this.storage.set('access_token', params.access_token);
-    this.storage.set('refresh_token', params.refresh_token);
     this.storage.set('expires_at', new Date().getTime() + expires * 1000);
```

Comment:
Is it still expiring that early ? ^^

Reply:
It's expiring after one hour but thanks to google auth and this login implementation it's not hard to receive a new token.